resources/views/settings/subscription/change.blade.php

```diff
@@ -208,28 +209,31 @@
             </p>
     @else
         {!! Form::open(['route' => ['settings.subscription.subscribe'], 'method' => 'POST', 'id' => 'cancellation-confirm', 'class' => 'subscription-form']) !!}
-
+        @php
+            $endDate = date($user->date_format, $user->upcomingInvoice()->period_end);
```

Comment:
I don't believe that `date()` uses the user's date format. Check how it's done for reminders, by searching for `dateRenderer` :)

Reply:
It does! I tried changing my date format and it works as intended for all the formats.